index.html

```diff
@@ -919,11 +945,100 @@ <h4>Verify Proof (eddsa-jcs-2022)</h4>
             </dd>
           </dl>
 
-          <p>
-The algorithm is the same as the one described in Section
-[[[#verify-proof-eddsa-rdfc-2022]]] with the changes outlined in
-Section [[[#eddsa-jcs-2022]]].
-          </p>
+          <ol class="algorithm">
+            <li>
+Let |unsecuredDocument| be a copy of |securedDocument| with
+the `proof` value removed.
+            </li>
+            <li>
+Let |proofOptions| be the result of a copy of |securedDocument|.|proof| with `proofValue`
+removed.
+            </li>
+            <li>
+Let |proofBytes| be the
+<a data-cite="VC-DATA-INTEGRITY#multibase-0">Multibase decoded base58-btc
+value</a> in |securedDocument|.|proof|.|proofValue|.
+            </li>
+            <li>
+Perform the following checks  between the |securedDocument|.<var>@context</var> and
+the |proofOptions|.<var>@context</var>:
```

Comment:
This should give us backwards compatibility.

Also note that if a holder deletes `proof.@context`, and `securedDocument.@context` has changed since the time of signing, then the signature will fail since the signed canonicalized proof will differ when `securedDocument.@context` is still copied over to `proof.@context` to generate the canonical proof to sign.

```suggestion
If |proofOptions|.<var>@context</var> exists, perform the following checks  between the |securedDocument|.<var>@context</var> the |proofOptions|.<var>@context</var>:
```

Reply:
The same protection should be afforded even if `@context` isn't copied to `proof` (for proof canonicalization) in this backwards compatibility case because the document signature would fail if its `@context` had changed. So the carve out @Wind4Greg mentions here: https://github.com/w3c/vc-di-eddsa/pull/79#discussion_r1589688204 should also work (and it seems work better).